source/MaterialXTest/MaterialXRender/RenderUtil.cpp

```diff
@@ -422,7 +422,7 @@ void ShaderRenderTester::addAdditionalTestStreams(mx::MeshPtr mesh)
 
     const std::string TEXCOORD_STREAM1_NAME("i_" + mx::MeshStream::TEXCOORD_ATTRIBUTE + "_1");
     mx::MeshFloatBuffer* texCoordData2 = nullptr;
-    if (!mesh->getStream(TEXCOORD_STREAM1_NAME))
+    if (!mesh->getStream(TEXCOORD_STREAM1_NAME, false))
```

Comment:
I like your decision to make `false` the default for allowing fallbacks in this method, as this is definitely the behavior users will expect.  Given that decision, though, I'd recommend that we omit the explicit argument when using the default behavior, as this seems more clear and compact.

Reply:
I'll fix this.